### ISSUE TITLE
10$: error on async/generator getter/setter

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1066]
The following code is invalid, but quick-lint-js does not complain:
```js
class C {
  async get foo() { return 42; }
}
```
